README.md

```diff
@@ -512,6 +570,40 @@ Contact us at Github Discussions: https://github.com/React-Automation-Studio/Rea
   The docker containers for RAS run in network  mode host. This is done to enable EPICS to communicate seamlessly with any IOC's on the same subnet as the host. Other OSes such as Windows may not support the host mode and will run in the bridged mode. This may break the communication between the micro services. It is therefore recommended to run the RAS containers on a Linux VM that is minimally running Ubuntu Server. Please ensure the the VM network interface is assigned an IP on the same subnet as your EPICS network in order for communication with the IOC's to occur seamlessly.
 
 # Changelog
+
+ V5.0.0 Friday 02 February 2024 Major Updates:
+  <ul>
```

Comment:
This is just a comment rather than a request. I am not sure why we keep using HTML tags for unordered lists in MD. I guess there are historical reasons. Yet, MD can do the same using "*". It feels to me, it is more readable in the plain text:
```
* Package updates to Backend modules
* Update to Epics 3 and Python 3.12.1
...
```

No need to make make change here, I am just curious.